pkg/operator/ceph/config/probes_test.go

```diff
@@ -51,26 +52,24 @@ func configLivenessProbeHelper(t *testing.T, keyType cephv1.KeyType) {
 		},
 	}
 	container := v1.Container{LivenessProbe: p}
-	l := map[cephv1.KeyType]*cephv1.ProbeSpec{keyType: {Disabled: true}}
-	type args struct {
-		daemon      cephv1.KeyType
-		container   v1.Container
-		healthCheck cephv1.CephClusterHealthCheckSpec
+	store := cephv1.ObjectStoreSpec{}
+	fs := cephv1.CephFilesystem{}
+	healthCheck := cephv1.CephClusterHealthCheckSpec{}
```

Comment:
For all these tests, there are no configured probes in `healthCheck` defined. As far as I can tell, none of the tests actually enabled. The only code paths that are being tested are "probe explicitly disabled" and "probe gets default value", but there is no test for if the probe gets an overridden value.